connectors/src/connectors/confluence/index.ts

```diff
@@ -349,6 +355,8 @@ export class ConfluenceConnectorManager extends BaseConnectorManager<null> {
           },
         });
 
+        await deleteFolderNode({ dataSourceConfig, folderId: internalId });
+
```

Comment:
@philipperolet would you rather delete from core first and then from connectors or the opposite? no strong opinion here